package.json

```diff
@@ -21,14 +21,14 @@
     "@oclif/test": "^2.2.20",
     "@salesforce/cli-plugins-testkit": "^3.2.12",
     "@salesforce/dev-config": "^3.1.0",
-    "@salesforce/dev-scripts": "^3.1.0",
+    "@salesforce/dev-scripts": "^4.0.0-beta.8",
```

Comment:
Are you wanting this prerelease @mshanemc

Reply:
yes, several are using it from here.  I kinda want some repos to be running through the cycle on wireit as "QA" before merging it to main.